medaka_bpm.py

```diff
@@ -24,14 +24,23 @@
 import src.setup as setup
 import src.segment_heart as segment_heart
 
+curr_dir = os.getcwd()
```

Comment:
cwd may not be the folder in which the file is located, but from whereever the process is started